java/src/jmri/jmrix/lenz/hornbyelite/EliteXNetTurnout.java

```diff
@@ -22,7 +22,7 @@ public EliteXNetTurnout(String prefix, int pNumber, XNetTrafficController tc) {
      * Send an "Off" message to the decoder for this output.
      */
     @Override
-    protected synchronized void sendOffMessage() {
+    protected synchronized void doSendOffMessage() {
```

Comment:
Send is a verb.  The do is redundant.

Reply:
pls. suggest a better naming; I did not come up with anything 'sane' and plain `sendOffMessage` was taken. I'm used to use this pattern, or e.g. `xxxProtected`, `performXXX`, but don't know what's the preference here.

Reply:
You renamed sendOffMessage as doSendOffMessage.  I would name it back.

Reply:
@pabender looking backwards after few days, I've outsmarted myself with the backward-compatibility-trampoline stuff - after the refactorings it does not bring any value.  Will revert back to just `sendOffMessage` as you suggest.